plugins/ssh-plugin/package.json

```diff
@@ -22,6 +22,9 @@
       "files": [
         "src"
       ],
+      "dependencies": {
+        "fs-extra": "7.0.1"
```

Comment:
Do we have CQ for `fs-extra`?

Reply:
There is a CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20011, BTW it is already used in che-theia https://github.com/eclipse/che-theia/search?q=fs-extra&unscoped_q=fs-extra